### PR TITLE
Port build processes over to voltrazord

### PR DESCRIPTION
**Pull request by contolini**
Generator changes per last week's pairing. Let me know how it works out.

### Added
- cf-type-testing to ignore list.

### Changed
- Migration to Voltrazord: Build processes now use npm instead of Bower.

### Fixed
- cf-theme-overrides.less now uses new brand palette color variables.

### Removed
- All Bower references.

## Review
- @Scotchester 

## Notes
- https://github.com/cfpb/capital-framework/pull/256 needs to be merged before this PR.

### CODE REVIEW

File: app/templates/grunt/_setup.sh [hunk 1]
```diff
@@ -6,26 +6,18 @@ set -e
...
 }
 
 # Clean project dependencies.
 clean(){
-  # If the node and bower directories already exist,
-  # clear them so we know we're working with a clean
-  # slate of the dependencies listed in package.json
-  # and bower.json.
-  if [ -d $NODE_DIR ] || [ -d $BOWER_DIR ]; then
+  # If the node directory already exist, clear it
```
Comment (Scotchester) [inline app/templates/grunt/_setup.sh]: *exists

File: app/templates/grunt/_setup.sh [hunk 2]
```diff
@@ -6,26 +6,18 @@ set -e
...
 # Clean project dependencies.
 clean(){
-  # If the node and bower directories already exist,
-  # clear them so we know we're working with a clean
-  # slate of the dependencies listed in package.json
-  # and bower.json.
-  if [ -d $NODE_DIR ] || [ -d $BOWER_DIR ]; then
+  # If the node directory already exist, clear it 
+  # so we know we're working with a clean slate of 
+  # the dependencies listed in package.json
```
Comment (Scotchester) [inline app/templates/grunt/_setup.sh]: Period at the end of the line.

**Comment by Scotchester** [comment]
Code looks good. Pulling down for a local test now.

File: app/templates/gulp/gulp/tasks/scripts.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ var browserSync = require( 'browser-sync' );
 gulp.task( 'scripts', function() {
   return gulp.src( config.src )
     .pipe( $.sourcemaps.init() )
-    .pipe( $.concat( config.name ) )
+    .pipe( $.webpack() )
```
Comment (Scotchester) [inline app/templates/gulp/gulp/tasks/scripts.js]: Ok, webpack is building a string of random characters for the filename of the final JS file, so it's not loading on the demo page. Need to keep `config.name` in here, I'm guessing.

**Comment by Scotchester** [comment]
Update to fix the earlier identified issue pushed. @contolini Can you pull this down and try a fresh generation and see if it works as expected? Also see if you run into that hanging issue.

**Comment by contolini** [comment]
Niiiiice! Both the grunt and gulp versions build without errors for me. And the generated dist directories look A-OK. Even the pesky icons are loading correctly.

**Comment by contolini** [comment]
Mergin' and publishin'.

**Comment by Scotchester** [comment]
Just to double-check: Do you mean that you ran the generator and it didn't hang just before running `setup.sh`?

**Comment by contolini** [comment]
Correct. :+1:

On Wednesday, January 6, 2016, Scott Cranfill <notifications@github.com>
wrote:

> Just to double-check: Do you mean that you ran the generator and it didn't
> hang just before running setup.sh?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cfpb/generator-cf/pull/107#issuecomment-169349781>.
>


-- 
Chris

**Comment by Scotchester** [comment]
Cool. Must've been a local issue for me.

**Pull request merged**